urnai/sc2/environments/sc2environment.py

```diff
@@ -0,0 +1,123 @@
+import sys
+
+from absl import flags
+from pysc2.env import sc2_env
+from pysc2.env.environment import TimeStep
+from pysc2.lib import actions, features
+
+from urnai.environments.environment_base import EnvironmentBase
+
+
+class SC2Env(EnvironmentBase):
+    def __init__(
+            self,
+            map_name='Simple64',
+            player_race=sc2_env.Race.random,
+            enemy_race=sc2_env.Race.random,
+            difficulty=sc2_env.Difficulty.very_easy,
+            visualize=False,
+            reset_done=True,
+            spatial_dim=16,
+            step_mul=16,
+            game_steps_per_episode=0,
+            realtime=False,
+            self_play=False,
+            action_space=actions.ActionSpace.RAW,
+            use_raw_units=True,
+            raw_resolution=64,
+            use_feature_units=True,
+            screen=64,
+            minimap=64,
+    ):
+        super().__init__(map_name, visualize, reset_done)
+
+        FLAGS = flags.FLAGS
+        FLAGS(sys.argv)
```

Comment:
```suggestion
       flags.FLAGS(sys.argv)
```

Reply:
I forgot to ask when I put this part of the code, but do you know what is the purpose of it? I tried to run the program without it and kept getting errors, but when I put these flags it worked normally. It was on the previous version of urnai, so I just copied it from there.

Reply:
I don't know what that part of the code is for. I recommend removing it and fixing any errors that appear.